src/app/routes/api/api.routes.ts

```diff
@@ -2,11 +2,13 @@ import { Router } from 'express'
 
 import { errorHandlerMiddlewares } from '../../loaders/express/error-handler'
 
+import { V1PlatformRouter } from './platform/v1'
 import { V1PublicRouter } from './public/v1'
 import { V3Router } from './v3'
 
 export const ApiRouter = Router()
 
 ApiRouter.use('/v3', V3Router)
 ApiRouter.use('/public/v1', V1PublicRouter)
+ApiRouter.use('/platform/v1', V1PlatformRouter)
```

Comment:
a quick clarification please - why is the new platform router not nested within the /v3 routes?
- is V1PublicRouter still in use or is this for backward compatibility?

Reply:
The new platform router is separate from the internal `api/v3/` routes, mainly because of version control and for the separate authentication mechanism.

V1PublicRouter is for the public APIs (non-platforms - means that admins are only allowed to carry out actions on forms that they own) - [here's](https://github.com/opengovsg/FormSG/pull/6376) the original PR. This route is not being used currently as we've yet to publicise it to admins (nor given anyone any API tokens).

V1PlatformRouter is for "platform" users - which gives them permission to carry out actions on other form admins' forms.

Hope that clarifies!